### ISSUE TITLE
Bug: configured prometheus backend not possible

### ISSUE THREAD

**Issue by BlackDark** [nautobot/helm-charts#306]
If you want to configure the prometheus engine for prometheus you get an error because it is not allowed by the schema.

https://github.com/nautobot/helm-charts/blob/fa0aadeec80d64e626cb59431f5b8d9ce7355749/charts/nautobot/values.schema.json#L297

**Comment by nniehoff** [comment]
Good catch I'll work on adjusting this.  However as a work around can you set metrics.enabled to true which will set the value for you
